functional-tests/utils.py

```diff
@@ -241,6 +241,20 @@ def check_submit_proof_fails_for_nonexistent_batch(seqrpc, nonexistent_batch: in
         raise AssertionError("Expected rpc error")
 
 
+def check_already_sent_proof(seqrpc, sent_batch: int):
+    """
+    This check requires that submitting proof that was already sent fails
```

Comment:
```suggestion
    Requires that submitting proof that was already sent fails
```

"This" and "check" are redundant. The function name is `check_foo`